doc/releases/changelog-dev.md

```diff
@@ -273,6 +273,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Fixes the matrices for qutrit operators when expanded to more wires.
```

Comment:
I think the updated method works for arbitrary spin-S not just qutrit right?

Reply:
updated the entry to reflect that it should work for higher dims.